tracing-attributes/src/attr.rs

```diff
@@ -177,7 +177,7 @@ impl Parse for EventArgs {
                     match ident.to_string().as_str() {
                         "Debug" => result.mode = FormatMode::Debug,
                         "Display" => result.mode = FormatMode::Display,
-                        "Raw" => result.mode = FormatMode::Raw,
+                        "Raw" => result.mode = FormatMode::Value,
                         _ => return Err(syn::Error::new(
                             ident.span(),
                             "unknown event formatting mode, expected either `Debug`, `Display`, or `Raw`",
```

Comment:
also, let's change "either" to "one of", since "either" generally refers to _two_ choices:
```suggestion
                            "unknown event formatting mode, expected one of `Debug`, `Display`, or `Raw`",
```

Reply:
Good call, I've updated this and the `err` case as well.